lib/commands/general.js

```diff
@@ -216,6 +221,111 @@ commands.closeApp = async function () {
   await this.adb.forceStop(this.opts.appPackage);
 };
 
+commands.getDisplayDensity = async function () {
+  let out = await this.adb.shell(['getprop', 'ro.sf.lcd_density']);
+  if (out) {
+    let val = parseInt(out, 10);
+    // if the value is NaN, try getting the emulator property
+    if (!isNaN(val)) {
+      return val;
+    } else {
```

Comment:
This `else` can be removed, now, since you're returning.

```js
let val = parseInt(out, 10);
if (!isNaN(val)) {
  return val;
}
// if the value is NaN, try getting the emulator property
log.debug(`Parsed density value was NaN: "${out}"`);
```